cli/cmd/vulnerability.go

```diff
@@ -270,6 +279,27 @@ func setDetailsFlag(cmds ...*flag.FlagSet) {
 	}
 }
 
+func setFailOnSeverityFlag(cmds ...*flag.FlagSet) {
+	for _, cmd := range cmds {
+		if cmd != nil {
+			cmd.StringVar(&vulCmdState.FailOnSeverity, "fail_on_severity", "",
+				fmt.Sprintf("specify a severity threshold to fail if vulnerabilities are found (%s)",
+					strings.Join(api.ValidEventSeverities, ", ")),
+			)
+		}
+	}
+}
+
+func setFailOnFixableFlag(cmds ...*flag.FlagSet) {
+	for _, cmd := range cmds {
+		if cmd != nil {
+			cmd.BoolVar(&vulCmdState.FailOnFixable, "fail_on_fixable", false,
+				"fail if the assessed container has fixable vulnerabilities (use in conjunction with --fail_on_severity)",
```

Comment:
Since this was very confusing internally, let's remove it until we find a better way to document that it is possible to use both flags together.
```suggestion
				"fail if the assessed container has fixable vulnerabilities",
```

Reply:
Will do!